spec/controllers/room_settings_controller_spec.rb

```diff
@@ -41,6 +41,7 @@
 
   describe '#update' do
     it 'uses MeetingOption::get_config_value and updates the setting if its config is "optional"' do
+      expect(MeetingOption).to receive(:get_config_value).and_call_original
```

Comment:
Is this required?

Reply:
It is required because `get_config_value` is called in the `after_create create_meeting_options` callback in Room.

Reply:
Um, I see what you mean.
However, I think we can further clean it.
If you intentionally wanna ensure that the method gets called twice you can assert that more specifically rather being so generic (that brakes a bit the resiliency of the tests).
Example instead of:
` expect(MeetingOption).to receive(:get_config_value).and_call_original`
This is more specific:
` expect(MeetingOption).to receive(:get_config_value).with(name: %w[glViewerAccessCode glModeratorAccessCode], provider: 'greenlight'| room.user.provider).and_call_original`
That ensures that if called that should be coming from the `Room#create_meeting_options`.
You can also stub the `Room#create_meeting_options` itself to isolate your test if it's implementation details are not relevant to this test. 
There's also a lot of what you can leverage from the Rspec mocking API that can better answer your need for this matter,